main.tf

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2018 Google LLC
```

Comment:
Should each of these be 2019 now?

Reply:
I suppose so. The templates haven't been, but I'll update it all.